lib/trino-plugin-toolkit/src/test/resources/file-based-system-access-table.json

```diff
@@ -11,7 +11,7 @@
     },
     {
       "user": "admin",
-      "privileges": ["SELECT", "INSERT", "DELETE", "OWNERSHIP"]
+      "privileges": ["SELECT", "INSERT", "UPDATE", "DELETE", "OWNERSHIP"]
```

Comment:
Why do we need to add "UPDATE" if user already has "OWNERSHIP"?

Reply:
FileBasedAccessControl#checkTablePermission doesn't give any special importance to OWNERSHIP privilege, it checks for the presence of specified privilege only.